### DISCUSSION TITLE
rehypeMathjax bundle size

### DISCUSSION THREAD

**Discussion by eliu-fracta** [remarkjs/remark#1225] [Q&A] [answered]
I'm using next13 and I'm wondering if there's a way to minizine the bundle size of mathjax
![image](https://github.com/remarkjs/remark/assets/109644973/51d0317e-d326-4514-b629-fb6409004565)

**Comment by wooorm** [comment, marked as answer]
mathjax is big, you choose to use it. Perhaps try KaTeX, it’s much smaller.

**Reply by edward1127** [reply]
thanks for your reply!
